php_zookeeper.c

```diff
@@ -1012,10 +1012,14 @@ void php_zk_watcher_marshal(zhandle_t *zk, int type, int state, const char *path
 #if PHP_VERSION_ID >= 70100
 #ifdef ZTS
 	(*cb_data->vm_interrupt) = 1;
+#else
+#if PHP_VERSION_ID >= 80200
+  EG(vm_interrupt) = (zend_atomic_bool){.value=1};
```

Comment:
Is there any possibility that this should be:
```c
zend_atomic_bool_store_ex(&EG(timed_out), true);
```

:)

Reply:
@falvarez ping

Reply:
@falvarez ping

Reply:
Did you compile it and did it work on PHP 8.2?